.github/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 2. Enhancement - Added UK Overseas Territory TACAN frequencies and coordinates as defined in UK MIL AIP - thanks to @chssn (Chris Parkinson)
 3. Bug - Corrected Duxford (EGSU) holding point and runway names - thanks to @chssn (Chris Parkinson)
 4. Bug - Removed files and references relating to Kinloss (EGQK) - thanks to @chssn (Chris Parkinson)
+5. Bug - Added Duxford (EGSU) ATZ - thanks to @chssn (Chris Parkinson)
```

Comment:
```suggestion
5. Enhancement - Added Duxford (EGSU) ATZ - thanks to @chssn (Chris Parkinson)
```
A bug is more where something is broken or doesn't work as intended, I would say this change was an enhancement since it's improving the sector file.